docs/docs/backdrop-filters.md

```diff
@@ -0,0 +1,89 @@
+---
+id: backdrops-filters
+title: Backdrop Filters
+sidebar_label: Backdrop Filters
+slug: /backdrops-filters
+---
+
+In Skia, backdrop filters are equivalent to their [CSS counterpart](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter). They allow you to apply image filters such as blurring to the area behind a [clipping mask](/docs/group#clipping-operations). A backdrop filter extends the [Group component](/docs/group#clipping-operations). All properties from the [group component](/docs/group) can be applied to a backdrop filter.
+
+## Backdrop Filter
+Applies an image filter to the area behind the whole canvas or behind a defined clipping mask. The first child of a backdrop filter is the image filter to apply. All properties from the [group component](/docs/group) can be applied to a backdrop filter.
```

Comment:
Should this "All properties from the [group component](/docs/group) can be applied to a backdrop filter." be removed since it is also in the above paragraph?

Reply:
That's fine - we'll keep it.